cubi-tools/update_workflow.py

```diff
@@ -14,7 +15,16 @@ def main():
     print(f"Project directory set as: {project_dir}")
     ref_repo_clone = args.ref_repo_clone
     ref_repo_curl = args.ref_repo_curl
-    ref_repo_wget = args.ref_repo_wget
+    branch = args.branch
+    ref_repo_wget = args.ref_repo_wget + branch + "/"
+
+    #detect if branch exists
+    response = requests.get(f"{ref_repo_wget}README.md")
+    if response.status_code != 200:
+        raise Exception("That is not a valid branch or tag")
+    else:
```

Comment:
totally unnecessary else here

Reply:
The whole check if the repo/branch exist has been replaced